src/controllers/userController/createNewUser.ts

```diff
@@ -0,0 +1,29 @@
+import { Request, Response } from 'express';
+import { User } from '../../models/user';
+import { CustomError } from '../../middlewares/globalErrorHandler';
+
+export const createNewUser = async (req: Request, res: Response) => {
+  const { userName, userEmail, firstName, lastName } = req.body;
+  try {
+    const user = await User.findOne({
+      where: {
+        userEmail,
+      },
+    });
+    if (user) {
+      console.log('user already exists>--------------------');
+      throw new CustomError('User already exists', 400);
+    }
+
+    const user1 = await User.create({
```

Comment:
why is this "user1" and not just "user"?

Reply:
I was using it in the same scope to check if the user already exists. I have updated it.